src/components/Project/PayProjectForm/PayProjectForm.tsx

```diff
@@ -34,13 +40,13 @@ export function PayProjectForm({ disabled }: { disabled?: boolean }) {
 
   return (
     <>
-      <div style={{ height: '22px' }}>
-        {error ? (
-          <span style={{ color: colors.text.failure, fontSize: '0.75rem' }}>
-            <Trans>Pay amount must be greater than 0.</Trans>
+      {errorMessage && (
+        <div style={{ height: '22px' }}>
+          <span style={{ color: colors.text.failure, fontSize: '0.7rem' }}>
```

Comment:
nitpick, for consistency, we shouldn't need a font size smaller than 0.75rem 🙏 

```suggestion
          <span style={{ color: colors.text.failure, fontSize: '0.75rem' }}>
```